pynumdiff/tests/test_diff_methods.py

```diff
@@ -48,7 +48,7 @@
                   [(-25, -25), (0, 0), (0, 0), (1, 1)],
                   [(-25, -25), (2, 2), (0, 0), (2, 2)],
                   [(-25, -25), (3, 3), (0, 0), (3, 3)]],
-    iterated_first_order: [[(-7, -7), (-10, -11), (0, -1), (0, 0)],
+    iterated_first_order: [[(-7, -7), (-9, -10), (0, -1), (0, 0)],
```

Comment:
These are what I had to loosen. Unfortunately I couldn't figure out which among the sea of numbers I had to increment, so now I `mark.parametrize` over the test functions too.